dashboard/app/models/levels/blockly.rb

```diff
@@ -562,4 +563,17 @@ def shared_functions
       SharedBlocklyFunction.where(level_type: type).map(&:to_xml_fragment)
     end.join
   end
+
+  # Display translated custom block text
+  def localized_shared_blocks(level_objects)
+    return level_objects if level_objects.blank?
```

Comment:
Small nit: we can return nil here because you're using `set_unless_nil` above.

```suggestion
    return if level_objects.blank?
```

Reply:
Ah yes, thanks!

Reply:
@joshlory The deletion of the blog configs are intentional.  These were untracked files from another branch I was working on.  I tried removing the files using rebase but it didn't work.